my-app/src/app/blog/carousel/carousel.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Component({
+  selector: 'carousel',
+  templateUrl: './carousel.component.html',
+  styleUrls: ['./carousel.component.css']
+})
+export class CarouselComponent implements OnInit {
+
+  constructor(private sanitizer: DomSanitizer) { }
+
+  ngOnInit() {
+    setInterval(() => {
+      this.onNext();
+    }, 2500)
+  }
+
+  selectedImageIndex = 0;
+  images: Array<any> =
+
+    [
+      { image: this.sanitizer.bypassSecurityTrustResourceUrl('https://www.facebook.com/plugins/post.php?href=https%3A%2F%2Fwww.facebook.com%2Fpermalink.php%3Fstory_fbid%3D775879029512636%26id%3D100012716381038&width=500') },
+      { image: this.sanitizer.bypassSecurityTrustResourceUrl('https://www.facebook.com/plugins/post.php?href=https%3A%2F%2Fwww.facebook.com%2FBDubb2x%2Fposts%2F2264839380494406&width=500') },
+      { image: this.sanitizer.bypassSecurityTrustResourceUrl("https://www.facebook.com/plugins/post.php?href=https%3A%2F%2Fwww.facebook.com%2Fyae.jones.9%2Fposts%2F2143381925767441&width=500") },
+      { image: this.sanitizer.bypassSecurityTrustResourceUrl("https://www.facebook.com/plugins/post.php?href=https%3A%2F%2Fwww.facebook.com%2FBigNisha%2Fposts%2F2505836309464590&width=500") },
+      { image: this.sanitizer.bypassSecurityTrustResourceUrl("https://www.facebook.com/plugins/post.php?href=https%3A%2F%2Fwww.facebook.com%2Fbrianna.everage%2Fposts%2F10158761789294698&width=500") },
+      { image: this.sanitizer.bypassSecurityTrustResourceUrl("https://www.facebook.com/plugins/post.php?href=https%3A%2F%2Fwww.facebook.com%2Fteontae.winters%2Fposts%2F2180003512064768&width=500") }
```

Comment:
Something that you'll run into a lot, is times where you'll have to type the same thing over and over again. We never want to do that. There's this principle called S.O.L.I.D. that people use to keep code maintainable and clean. The D stands for DRY, (D)on't (R)epeat (Y)ourself. Let's try to not repeat ourselves here by making an array and mapping each url to use the ```bypassSecurityTrustResourceUrl``` method.